ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -10,6 +10,30 @@
 }
 @using Microsoft.AspNetCore.Http.Extensions @* for GetEncodedUrl *@
 
+@{
+    var mailtoUrl = "mailto:";
+
+    // to: Add the proper support agent as recipient
+    mailtoUrl += Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com";
+    // cc: Cc the team's email address because what if this email account is a random personal one?
+    if (Model.Team.PrimaryContactEmail != null)
+    {
+        mailtoUrl += "?cc=" + Uri.EscapeDataString(Model.Team.PrimaryContactEmail);
+    }
+
+    // subject: Make this be about this puzzle
+    mailtoUrl += "&subject=" + Uri.EscapeDataString("[" + Model.Event.Name + "]");
+    // subject: Make this be from this team
+    mailtoUrl += Uri.EscapeDataString(" [" + Model.Team.Name + "]");
+
+    // request out of email mode
+    // subject: Add the Email Mode signifier
+    mailtoUrl += Uri.EscapeDataString(" [⚠ Team Disqualified ⚠]");
+    // body: Invite solver to give details
+    mailtoUrl += "&body=" + Uri.EscapeDataString(
+        "Use this email to fully explain your thought process so we know why you would like your team disqualification to be undone! 😇" + Environment.NewLine + Environment.NewLine);
```

Comment:
"Use this mail to explain anything you think we need to know." As written, it sounds like we'll happily undisqualify you for any reason...